docs/testhelpers.rst

```diff
@@ -1,9 +1,11 @@
 .. _testhelpers:
 
-************************************************
-Astropy Testing helpers (`astropy.tests.helper`)
-************************************************
+***************
+Astropy Testing
```

Comment:
Maybe "Astropy Testing Tools" or "Astropy Testing Machinery"?  That would make it clearer that this is the machinery, while the guidelines are below.

Along the same lines, perhaps under this title (but before the helper module heading), you should add a sentence with a link to the testing guidelines.  Something like ``This section is primarily a reference for developers that want to understand or add to the Astropy testing machinery.  See :doc:/development/testguide for an overview of running or writing the tests.``